envman/run_test.go

```diff
@@ -0,0 +1,87 @@
+package envman
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/bitrise-io/envman/models"
+)
+
+func TestExpandEnvsInString(t *testing.T) {
+	t.Log("TestExpandEnvsInString")
+
+	inp := "${HOME} home"
+	expanded := expandEnvsInString(inp)
+
+	home := os.Getenv("HOME")
```

Comment:
also test with a custom ENV, one you set for yourself before calling expandEnvs...